Source/WebKit/UIProcess/Cocoa/XPCConnectionTerminationWatchdog.h

```diff
@@ -52,9 +52,7 @@ class XPCConnectionTerminationWatchdog {
 
     OSObjectPtr<xpc_connection_t> m_xpcConnection;
     RunLoop::Timer m_watchdogTimer;
-#if PLATFORM(IOS_FAMILY)
```

Comment:
Should this be `USE_RUNNINGBOARD`? We don't use assertions on downlevels (e.g. for macOS < 13.3).